src/transform/filter/filterByIsbnIdentifier.js

```diff
@@ -4,23 +4,30 @@ import ConversionError from '../convert/conversionError.js';
  * Filter filtering items without isbn identifiers.
  * @returns Object containing filter and its name
  */
-export function filterByIsbnIdentifier(filterConfig) {
-  const reversed = filterConfig?.filterByIsbnIdentifier?.reverse;
-
-  return {
-    filter,
-    name: 'filterByIsbnIdentifier'
-  };
+export function filterByIsbnIdentifier({active = true, reverse = false}) {
```

Comment:
Function parameters need to be documented. Reverse might not be obvious at first glance.